application-form.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Simple HTML Form</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="validator.js"></script>
+	
+</head>
+<body>
+<script>
+function HidePassword() { 
```

Comment:
Convention in JS function names to be in lower-uppper case. In this case "hidePassword" would be appropriate.

Reply:
Okay, I'll remake that.

Reply:
I remade it.